app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -77,9 +77,9 @@ fun ButtonBar() {
 }
 
 @Composable
-fun Menu() {
+fun menu() {
     val items = listOf("A", "B", "C", "D", "E", "F")
-    var countryName : String by remember { mutableStateOf("Menu Categories") }
+    var menuName : String by remember { mutableStateOf("Menu Categories") }
```

Comment:
changed countryName to menuName to go along with the project. I believe this variable was leftover from another project.